website/static/js/projectSettings.js

```diff
@@ -107,7 +107,40 @@ var ProjectSettings = oop.extend(
                     }
                 }
             };
-        }
+        },
+        enableMailingList: function() {
+            var request = $.ajax({
+                type: 'post',
+                dataType: 'json',
+                url: nodeApiUrl + 'mailing_list/'
+            });
+            request.done(function() {
+                window.location.reload();
+            });
+            request.fail($osf.handleJSONError);
+        },
+        disableMailingList: function(nodeType) {
+
+            var message = '<p>Are you sure that you want to disable the mailing list for this ' + nodeType + '?';
```

Comment:
![screen shot 2016-02-29 at 9 32 13 am](https://cloud.githubusercontent.com/assets/134455/13397309/5ad3856e-dec7-11e5-9dcd-7d52957d968f.png)

This `select` is [wired up](https://github.com/CenterForOpenScience/osf.io/blob/af226bba8d0f9159151aafcdb0e69da4cd9d67d5/website/static/js/pages/project-settings-page.js#L41-L57) in `project-settings-page.js`, which delegates to a [`ProjectMailingList`](https://github.com/CenterForOpenScience/osf.io/blob/af226bba8d0f9159151aafcdb0e69da4cd9d67d5/website/static/js/mailingListTreebeard.js#L76-L163), which calls out to [`subscribe`](https://github.com/CenterForOpenScience/osf.io/blob/af226bba8d0f9159151aafcdb0e69da4cd9d67d5/website/static/js/mailingListTreebeard.js#L43-L59) for the AJAX call.